### ISSUE TITLE
Missing schedule_storage/alembic/ in dagster_postgres?

### ISSUE THREAD

**Issue by alirazeen** [dagster-io/dagster#2601]
I was testing the `dagster instance migrate` command, to make sure database migrations are applied correctly, when I ran into the following error. I'm using v0.8.1:

```
(venv)$ dagster instance migrate
$DAGSTER_HOME: /opt/dagit

Updating run storage...
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
Updating event storage...
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
Updating schedule storage...
Traceback (most recent call last):
  File "/opt/dagit/venv/bin/dagster", line 8, in <module>
    sys.exit(main())
  File "/opt/dagit/venv/lib/python3.6/site-packages/dagster/cli/__init__.py", line 38, in main
    cli(obj={})  # pylint:disable=E1123
  File "/opt/dagit/venv/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/opt/dagit/venv/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/opt/dagit/venv/lib/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/dagit/venv/lib/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/dagit/venv/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/dagit/venv/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/opt/dagit/venv/lib/python3.6/site-packages/dagster/cli/instance.py", line 49, in migrate_command
    instance.upgrade(click.echo)
  File "/opt/dagit/venv/lib/python3.6/site-packages/dagster/core/instance/__init__.py", line 406, in upgrade
    self._schedule_storage.upgrade()
  File "/opt/dagit/venv/lib/python3.6/site-packages/dagster_postgres/schedule_storage/schedule_storage.py", line 85, in upgrade
    run_alembic_upgrade(alembic_config, self._engine)
  File "/opt/dagit/venv/lib/python3.6/site-packages/dagster/core/storage/sql.py", line 27, in run_alembic_upgrade
    upgrade(alembic_config, rev)
  File "/opt/dagit/venv/lib/python3.6/site-packages/alembic/command.py", line 278, in upgrade
    script = ScriptDirectory.from_config(config)
  File "/opt/dagit/venv/lib/python3.6/site-packages/alembic/script/base.py", line 149, in from_config
    hook_config=config.get_section("post_write_hooks", {}),
  File "/opt/dagit/venv/lib/python3.6/site-packages/alembic/script/base.py", line 70, in __init__
    "scripts folder." % os.path.abspath(dir)
alembic.util.exc.CommandError: Path doesn't exist: '/opt/dagit/venv/lib/python3.6/site-packages/dagster_postgres/schedule_storage/alembic'.  Please use the 'init' command to create a new scripts folder.
```

It looks like `schedule_storage/alembic` is missing from the `dagster-postgres` package because it's not included in `MANIFEST.in`:

https://github.com/dagster-io/dagster/blob/d71543542ffd054859fa6b16920fb46df83ceaf6/python_modules/libraries/dagster-postgres/MANIFEST.in#L1-L3

Is this a bug? Or should I be running migrations differently when I'm using postgres for schedule storage?

**Comment by alangenfeld** [comment]
good catch and thanks for the thorough report well get this fix out asap

**Comment by astenuz** [comment]
I'm having the same issue, how did you fix it to make the migration work?

**Comment by alangenfeld** [comment]
we will push out the fix in `0.8.2` later today. In the mean time I believe `dagster schedule wipe` will put you in a state where you can proceed.
